src/mumble/MainWindow.cpp

```diff
@@ -564,6 +562,14 @@ void MainWindow::updateTransmitModeComboBox() {
 	}
 }
 
+QMenu *MainWindow::createPopupMenu() {
+	if (g.s.wlWindowLayout == Settings::LayoutCustom) {
+		return QMainWindow::createPopupMenu();
+	}
+
+	return new QMenu();
```

Comment:
I think this might leak. Following the documentation (http://doc.qt.io/qt-4.8/qmainwindow.html#createPopupMenu) I simply return NULL instead which prevents a menu from showing up at all.